Retrovizor.Domain/Repositories/Implementations/ReviewRepository.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Retrovizor.Data.Entities;
+using Retrovizor.Data.Entities.Models;
+using Retrovizor.Domain.Repositories.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Retrovizor.Domain.Repositories.Implementations
+{
+    public class ReviewRepository : IReviewRepository
```

Comment:
Mislim da bi trebala postojat mogucnost uredivanja i brisanja review-a.

Reply:
U pravu si ja san gleda iz admin perspektive al polazniku triba to omogucit. Will fix.